modules/aws/eks/variables.tf

```diff
@@ -63,6 +68,7 @@ variable "map_users" {
     username = string
     groups   = list(string)
   }))
+  default = []
```

Comment:
why is this being introduced?

Reply:
when doing a local testing, i was getting an error. Just added a default .